### PR TITLE
Adds icons for items with randomization

### PR DESCRIPTION
**Pull request by ShizCalev**
Adds icons for items that have randomized features (ie change colors, subtypes, ect) when they spawn so they are more easily distinguished from normal versions in DM.

These are not visible ingame.

Also fixes invisible engineering scanners and headphones in DM.

image to confirm nothing was messed up ingame.
image

DM example;
image

### CODE REVIEW

**Comment by Bawhoppen** [comment]
The jumpsuit one looks kind of like the rainbow jumpsuit

**Comment by Partheo** [comment]
Why only add rainbow to some things and question marks to others? Just stick with a standard- I suggest a question mark for every random item. Rainbows are pretty, but they don't matter because they won't show up in game.

**Comment by ShizCalev** [comment]
Rainbows were for things that only change color, question marks were for things that change attributes beyond color.

**Comment by Partheo** [comment]
Alright, well at least you did have some standard, which is good. But yeah, questions marks on everything would be good.

File: code/game/objects/items/devices/flashlight.dm [hunk 1]
```diff
@@ -488,16 +488,14 @@
...
-	var/list/glowtypes = typesof(/obj/item/device/flashlight/glowstick)
-	glowtypes -= /obj/item/device/flashlight/glowstick/random
-
-	var/obj/item/device/flashlight/glowstick/glowtype = pick(glowtypes)
-
-	name = initial(glowtype.name)
-	color = initial(glowtype.color)
 	. = ..()
+	var/T = pick(typesof(/obj/item/device/flashlight/glowstick) - /obj/item/device/flashlight/glowstick/random)
+	new T(get_turf(src))
```
Comment (Cyberboss) [inline code/game/objects/items/devices/flashlight.dm]: use loc for transformations

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**